src/realm/sync/version.hpp

```diff
@@ -6,7 +6,7 @@
 // clang-format off
 #define REALM_SYNC_VER_MAJOR 10
 #define REALM_SYNC_VER_MINOR 0
-#define REALM_SYNC_VER_PATCH 0-beta.4
+#define REALM_SYNC_VER_PATCH 0-beta.9
```

Comment:
Is that right?

Reply:
This has no implications. The information will be updated anyway when we are to make a new release